src/main/java/marites/Marites.java

```diff
@@ -0,0 +1,312 @@
+        case COMMAND_DELETE:
+            commandOutput = executeDelete(command);
+            break;
+        default:
+            commandOutput = UNKNOWN_COMMAND_MESSAGE;
+            break;
+        }
+        return commandOutput;
+    }
+
+    /*
+        The following methods take in a String, denoting a command.
+        They return a String, denoting the feedback after executing the command.
+     */
+
+    /**
+     * Executes a list command.
+     * @return The list of tasks.
+     */
+    private static String executeListTasks() {
+        StringBuilder taskList = new StringBuilder();
+        for (int i = 1; i <= tasks.size(); ++i) {
+            String task = String.format(LIST_TASK_ITEM_FORMAT_STRING, i, tasks.get(i-1));
+            taskList.append(task);
+        }
+        return taskList.toString();
+    }
+
+    /**
+     * Parses an add task command given by the user.
+     * @param taskType The task's type.
+     * @param command The user's command.
+     * @return A marites.task.Task object representing the task.
+     */
+    private static Task parseAddTask(String taskType, String command)
+        throws EmptyTaskDescriptionException, MissingParameterException, UnknownTaskTypeException {
+        String[] parametersSplit;
+        if (command.length() == 0) {
+            throw new EmptyTaskDescriptionException();
+        }
+        switch (taskType) {
+        case COMMAND_ADD_TODO:
+            return new Todo(command);
+        case COMMAND_ADD_DEADLINE:
+            parametersSplit = command.split(COMMAND_ADD_DEADLINE_TAG);
+            if (parametersSplit.length == 1) {
+                throw new MissingParameterException(COMMAND_ADD_DEADLINE_TAG);
+            }
+            return new Deadline(parametersSplit[0].strip(), parametersSplit[1].strip());
+        case COMMAND_ADD_EVENT:
+            parametersSplit = command.split(COMMAND_ADD_EVENT_TAG);
+            if (parametersSplit.length == 1) {
+                throw new MissingParameterException(COMMAND_ADD_EVENT_TAG);
+            }
+            return new Event(parametersSplit[0].strip(), parametersSplit[1].strip());
+        default:
+            throw new UnknownTaskTypeException(taskType);
+        }
+    }
+    /**
+     * Executes an add task command.
+     * @param taskType The task's type.
+     * @param command The user's command.
+     * @return A feedback string with the added task.
+     */
+    private static String executeAddTask(String taskType, String command) {
+        Task task;
+        try {
+            task = parseAddTask(taskType, command);
+        } catch (MissingParameterException e) {
+            return String.format(ADD_TASK_MISSING_PARAMETER_MESSAGE, e.getMissingParameterTag());
+        } catch (EmptyTaskDescriptionException e) {
+            return ADD_TASK_MISSING_DESCRIPTION_MESSAGE;
+        } catch (UnknownTaskTypeException e) {
+            return String.format(ADD_TASK_UNKNOWN_TASK_MESSAGE, e.getTaskType());
+        }
```

Comment:
You might want to figure out how to better implement the exceptions handling in this section.